src/validationHelper.ts

```diff
@@ -26,10 +26,30 @@
 
 module powerbi.extensibility.utils.dataview {
     export module validationHelper {
-        export function checkIsImageUrlAllowable(url: string): boolean {
-            // Excludes all URLs that don't contain .gif .jpg .png or .svg extensions.
-            // Also excludes directives "javascript:" and "data:".
-            return (/\.(gif|jpg|png|svg)$/i).test(url) && !(/(javascript:|data:)/i).test(url);
+        export function isImageUrlAllowed(url: string): boolean {
+            // Excludes all URLs that don't contain .gif .jpg .png or .svg extensions and don't start from "http(s)://".
+            // As a result -- also excludes all directives such as "javascript:", "data:" and "blob:".
+            return (/^https?:\/\/.+\.(gif|jpg|png|svg)$/i).test(url);
```

Comment:
I'm wondering why this RegExp excludes base64 encoded images?

Reply:
Do we really need base64 images?